packages/material-ui/src/IconButton/IconButton.test.js

```diff
@@ -104,15 +102,4 @@ describe('<IconButton />', () => {
       expect(button).to.have.class(classes.disabled);
     });
   });
```

Comment:
```suggestion
  });
  
  it('should raise a warning about onClick in children because of Firefox', () => {	
    expect(() => {	
      PropTypes.checkPropTypes(	
        IconButton.propTypes,	
        { classes: {}, children: <svg onClick={() => {}} /> },	
        'prop',	
        'MockedName',	
      );	
    }).toErrorDev(['Material-UI: You are providing an onClick event listener']);	
  });
```

Reply:
I guess the only problem was using `Naked` which was previously coming from `withStyles`. It should work like this

Reply:
Ah, I misread the commit for `Accordion` to remove `Naked`.  My bad!